pytorch_tabnet/multiclass_utils.py

```diff
@@ -26,17 +26,21 @@ def _assert_all_finite(X, allow_nan=False):
     # everything is finite; fall back to O(n) space np.isfinite to prevent
     # false positives from overflow in sum method. The sum is also calculated
     # safely to reduce dtype induced overflows.
```

Comment:
This has to be check but I think this file is actually never used in the repo, if it's correct we should probably remove it.

Reply:
`infer_multitask_output` is still used into `multitask.py` and the function use all the functions from this file

Reply:
yep you're right!